gtfs/gtfs_utils/gtfs_utils/local_files.py

```diff
@@ -18,20 +19,48 @@ def _get_existing_output_files(output_folder: str) -> List[Tuple[datetime.date,
     configuration = load_configuration()
     file_name_re = configuration.files.output_file_name_regexp
     file_type_re = configuration.files.output_file_type.replace('.', '\\.')
-    regexp = file_name_re + '\\.' + file_type_re
+    regexp = re.compile(file_name_re + '\\.' + file_type_re)
+
+    # validate that the regex used the correct group names
+    faulty_group_names = False
+    if "type" in regexp.groupindex and "date_str" in regexp.groupindex:
```

Comment:
Wouldn't you like to put it inside `_parse_file_name_regex_match`?

Reply:
I thought about that but than the log message would log for every file, although they are all using the same regex.
I thought creating a global variable just for that is a bit disgusting.